text/0232-maturity-of-otel.md

```diff
@@ -0,0 +1,151 @@
+
+Projects MUST list their Tier-1 Components in the main readme file. A Tier-1 Component MAY be hosted under the project's "contrib" repository. Not all components under the main repository (sometimes called the "core" repository) are automatically Tier-1 Components.
+
+A component can be declared "Tier-1" only if they have the following characteristics:
+
+* The component has at least the same maturity level as the SIG
+* The component has a clear code owner
+* The code owner is active within the project
+* The maintainers of the project are comfortable supporting the component should the code owner decide to step down or leave the project
+
+### Tier-1 Signals
+
+Similar to a Tier-1 Component, a Tier-1 Signal must have stable support in order to declare OpenTelemetry as stable. Declaring a signal as Tier-1 means that all projects must also see that signal as required for declaring the project stable.
+
+Not all signals will have the same requirements. For instance, Logs might not provide an instrumentation API specification.
+
+List of Tier-1 signals and their capabilities:
+
+* Logs
+  * Data model
+  * SDK specification
+  * Instrumentation SDKs
+  * Bridge API specification
+* Metrics
+  * Data model
+  * API specification
+  * Instrumentation APIs
+  * SDK specification
+  * Instrumentation SDKs
+* Traces
+  * Data model
+  * API specification
+  * Instrumentation APIs
+  * SDK specification
+  * Instrumentation SDKs
+
+The lifecycle of a signal is defined in the [Versioning and stability for OpenTelemetry clients](https://opentelemetry.io/docs/reference/specification/versioning-and-stability). While the stability and versioning of the signals may relate to this OTEP, the previously linked document specifies the details on how to reach the "stable" level mentioned in this OTEP.
+
+The "experimental" level from the signal versioning and stability document relates to the "alpha" and "beta" levels from this OTEP, "stable" and "deprecated" are equivalent to their counterparts on this document, and "removed" is non-existent here.
+
+When a new signal reaches the "stable" level, OpenTelemetry TC and GC will ultimately decide whether to include it as part of OpenTelemetry's priority components, using the process described in [Promoting SIGs and components](#promoting-sigs-and-components).
+
+### Promoting SIGs and components
+
+SIGs can request to be included in the Tier-1 List by contacting a TC or GC member. The TC/GC member will bring the request to the TC/GC via Slack or Zoom call, who will vote on the matter. A simple majority suffices for including the SIG. The SIG needs to have at least equal maturity as the project. For instance, if OpenTelemetry's maturity is set to Beta, the coming SIG has to be at least Beta as well, so that it doesn't downgrade the main project's maturity.
```

Comment:
> A simple majority suffices for including the SIG.

Typo?

Reply:
Not sure what I meant by this. Maybe was reading too fast originally? Anyway, closing.